requirement_data.js

```diff
@@ -437,7 +422,7 @@ const requirement_data = {
             new Requirement(5, [[Moves.Barrels]]), // 1 bunch in CrystalCavesMain
             new Requirement(11, [[Moves.CavesIceWalls]]), // 1 bunch, 3 bananas in CavesGGRoom; 3 bananas in CavesSnideArea
             new Requirement(6, [[Moves.Barrels, Moves.CavesIceWalls]]), // 6 bananas in BoulderCave
-            new Requirement(10, [[Moves.Mini, Moves.Pineapple]]), // 1 balloon in CavesBonusCave
+            new Requirement(10, [[Moves.Mini, Moves.Pineapple, Moves.W3Bonus]]), // 1 balloon in CavesBonusCave
```

Comment:
Can you please explain what W3Bonus is? I wasn't able to derive it from the logic.